playbooks/ansible/tower/update-tower-cert.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
@bvkin this playbook should re-use the existing role, rather than re-implement the role

Reply:
Basically split the [following section [1]](https://github.com/redhat-cop/infra-ansible/blob/master/roles/ansible/tower/config-ansible-tower/tasks/install.yml#L43-L56) out into its own file and include it this playbook


[1] : https://github.com/redhat-cop/infra-ansible/blob/master/roles/ansible/tower/config-ansible-tower/tasks/install.yml#L43-L56